src/components/Sidebar/index.jsx

```diff
@@ -35,6 +35,14 @@ import RustIcon from './RustIcon';
     color: theme.palette.grey[800],
     fontFamily: 'Roboto500',
   },
+  link: {
+    position: 'absolute',
+    display: 'block',
+    bottom: 0,
```

Comment:
I think we can add padding so that the links are aligned vertically with the Skills.
```suggestion
    padding: 3 * theme.spacing.unit,
```

After:
<img width="242" alt="Screen Shot 2019-08-22 at 11 46 49 PM" src="https://user-images.githubusercontent.com/24544912/63529605-806cd500-c537-11e9-8d68-14d2765f6869.png">